public/app/features/alerting/unified/components/rules/RuleListStateView.tsx

```diff
@@ -71,7 +78,7 @@ const STATE_TITLES: Record<PromAlertingRuleState, string> = {
 const RulesByState = ({ state, rules }: { state: PromAlertingRuleState; rules: CombinedRule[] }) => {
   const { page, pageItems, numberOfPages, onPageChange } = usePagination(rules, 1, DEFAULT_PER_PAGE_PAGINATION);
 
-  const isNotFiringState = state !== PromAlertingRuleState.Firing;
+  const isFiringState = state !== PromAlertingRuleState.Firing;
```

Comment:
shouln't be `isNotFiringState`?

Reply:
Good point - this was reverted back to the previous (inaccurate) name